src/views/undo.h

```diff
@@ -20,25 +19,16 @@
 #define DT_UNDO_H
 
 #include "common/dtpthread.h"
-#include "views/view.h"
 #include <glib.h>
 
 //  types that are known by the undo module
 typedef enum dt_undo_type_t
 {
-  DT_UNDO_GEOTAG = 1,
+  DT_UNDO_GEOTAG = 1 << 0,
+  DT_UNDO_HISTORY = 1 << 1,
+  DT_UNDO_ALL = DT_UNDO_GEOTAG | DT_UNDO_HISTORY
 } dt_undo_type_t;
```

Comment:
Eventually I would like to get rid of `dt_undo_type_t` as that hardcodes the possible undo users in the core. But that is something for the future, for now I'm fine with it.